README.md

```diff
@@ -1,55 +1,140 @@
 # Bottom-up conditioned top-down pose estimation (BUCTD) 
 
 [![PWC](https://img.shields.io/endpoint.svg?url=https://paperswithcode.com/badge/rethinking-pose-estimation-in-crowds/pose-estimation-on-crowdpose)](https://paperswithcode.com/sota/pose-estimation-on-crowdpose?p=rethinking-pose-estimation-in-crowds)
-[![PWC](https://img.shields.io/endpoint.svg?url=https://paperswithcode.com/badge/rethinking-pose-estimation-in-crowds/animal-pose-estimation-on-marmoset-8k)](https://paperswithcode.com/sota/animal-pose-estimation-on-marmoset-8k?p=rethinking-pose-estimation-in-crowds)
 
-This repository will contain the official code for our pre-print: [Rethinking pose estimation in crowds: overcoming the detection information-bottleneck and ambiguity](https://arxiv.org/abs/2306.07879).
+This repository contains the official code for our paper: [Rethinking pose estimation in crowds: overcoming the detection information-bottleneck and ambiguity](https://openaccess.thecvf.com/content/ICCV2023/papers/Zhou_Rethinking_Pose_Estimation_in_Crowds_Overcoming_the_Detection_Information_Bottleneck_ICCV_2023_paper.pdf). 
+[[YouTube Video]](https://www.youtube.com/watch?v=BHZnA-CZeZY) [[Website]](https://amathislab.github.io/BUCTD/)
 
-- This work is accepted to ICCV 2023 🎉
 
-- This work was also presented at the 2023 [CV4Animals workshop at CVPR](https://www.cv4animals.com)
+- Sep 2023: We released the code :)
+- July 2023: This work is accepted to ICCV 2023 🎉
+- June 2023: BUCTD was also presented at the 2023 [CV4Animals workshop at CVPR](https://www.cv4animals.com)
+- June 2023: An earlier version can be found on [arxiv](https://arxiv.org/abs/2306.07879)
 
-![BUCTD](media/BUCTD_fig1.png)
-
-- Stand-alone code will be made available by ICCV (Oct 2023)!
+<img src="media/BUCTD_fig1.png" width="600">
 
 - This code will also be integrated in [DeepLabCut](https://github.com/DeepLabCut/DeepLabCut)!
 
-### Code Acknowledgements
+### Installation 
+
+We developed and tested our models under ```python=3.8.10, pytorch=1.8.0, cuda=11.1```. Other versions might be working as fine.
+
+<details>
+  <summary>Instructions</summary>
+   
+   1. Clone this repo, and we'll call the directory that you cloned as ${BUCTD_ROOT}.
+
+   ```sh
+   git clone https://github.com/amathislab/BUCTD.git
+   cd ${BUCTD_ROOT}
+   ```
+
+   2. Install Pytorch and torchvision
+
+   Follow the instruction on https://pytorch.org/get-started/locally/.
+   ```sh
+   # an example:
+   conda install pytorch==1.8.0 torchvision==0.9.0 cudatoolkit=11.1 -c pytorch -c conda-forge
+   ```
+
+   3. Install other needed packages
```

Comment:
grammar

Reply:
@LucZot this isn't fixed "install other needed packages --> "Install additional dependencies"